spec/system/composer/discard_draft_spec.rb

```diff
@@ -1,68 +1,72 @@
 # frozen_string_literal: true
 
 describe "Composer - discard draft modal", type: :system do
-  fab!(:current_user) { Fabricate(:admin) }
-  fab!(:topic_1) { Fabricate(:topic) }
+  fab!(:topic)
+  fab!(:user) { Fabricate(:admin) }
 
   let(:topic_page) { PageObjects::Pages::Topic.new }
-  let(:discard_draft_modal) { PageObjects::Modals::DiscardDraft.new }
   let(:composer) { PageObjects::Components::Composer.new }
+  let(:discard_draft_modal) { PageObjects::Modals::DiscardDraft.new }
 
-  before { sign_in(current_user) }
+  before { sign_in(user) }
 
   context "when editing different post" do
-    fab!(:post_1) { Fabricate(:post, topic: topic_1, user: current_user) }
-    fab!(:post_2) { Fabricate(:post, topic: topic_1, user: current_user) }
+    fab!(:post_1) { Fabricate(:post, topic:, user:) }
+    fab!(:post_2) { Fabricate(:post, topic:, user:) }
```

Comment:
Curious, what does `topic:, user:` do here?

Reply:
Since the variables have the same name, you can omit them in "new-ish" Ruby ;) Less typing 🙌

Reply:
TIL! similar to JS, but different enough to be confusing!